### DISCUSSION TITLE
"Repeated" field drops errors to browser console and don't work

### DISCUSSION THREAD

**Discussion by Pos3iD1** [laravel/nova-issues#5718] [Ideas & Feature Requests] [answered]
- Laravel Version: 10.15.0
- Nova Version: 4.26.4
- PHP Version: 8.2.8
- Database Driver & Version: mysql  Ver 8.0.31-0ubuntu0.20.04.2 for Linux on x86_64 ((Ubuntu))
- Operating System and Version: Ubuntu 22.04.2 LTS
- Browser type and version: FireFox, Chrome
- Reproduction Repository: Private, Bitbucket

### Description:
When I added new Beta Repeater field to Action fields list and tried to add line in this repeater when starting action it drops errors to browser console and don't work
![image](https://github.com/laravel/nova-issues/assets/73269214/90f951f5-8ba6-497f-b300-e3ffdad71078)

### Detailed steps to reproduce the issue on a fresh Nova installation:
1. Create new Repeater with some fields
2. Add created Repeater to an Action fields
3. Try to add line to repeater in Action pop-up fields

**Comment by crynobone** [comment, marked as answer]
Repeater field doesn't support Action Field at the moment and wasn't designed for it. Moving this to Idea and Feature Request.

**Comment by Livijn** [comment]
I would really appreciate support for using the Repeater in Actions :)
